[3.7] docs(date pickers): OnCalendarCellRender event (#1182)
telerik/blazor-docs: components/calendar/events.md

The event arguments are of type `CalendarCellRenderEventArgs` and provide the following fields:

| Property | Type | Description |
| --- | --- | --- |

> | `Class` | `string` | Lets you set a custom CSS class to the cell DOM element. |

Reviewer: ```suggestion
| `Class` | `string` | A custom CSS class for the cell DOM element. |
```

| `Date` | `DateTime` | The date of the cell |

> | `View` | `CalendarView` enum <br /> `Month` | The currently visible view. You can use it to determine if the calendar is rendering the MonthView, YearView, and so on. |

Reviewer: We normally use brackets for the default value.

```suggestion
| `View` | `CalendarView` enum <br /> (`Month`) | The currently visible view. You can use it to determine if the calendar is rendering the MonthView, YearView, and so on. |
```
